topics/sequence-analysis/tutorials/quality-control/tutorial.md

```diff
@@ -170,140 +157,297 @@ To estimate sequence quality along all sequences, we now use [FastQC](https://ww
+
+It is usually the case for RNA sequencing where there is some very highly abundant transcripts and some lowly abundant. It is expected that duplicate reads will be observed for high abundance transcripts:
+
+![Sequence Duplication Levels for RNA-seq](../../images/quality-control/sequence_duplication_levels_rna_seq.png)
+
+## Overrepresented sequences
+
+A normal high-throughput library will contain a diverse set of sequences, with no individual sequence making up a tiny fraction of the whole. Finding that a single sequence is very overrepresented in the set either means that it is highly biologically significant, or indicates that the library is contaminated, or not as diverse as you expected.
+
+FastQC lists all of the sequence which make up more than 0.1% of the total. For each overrepresented sequence FastQC will look for matches in a database of common contaminants and will report the best hit it finds. Hits must be at least 20bp in length and have no more than 1 mismatch. Finding a hit doesn't necessarily mean that this is the source of the contamination, but may point you in the right direction. It's also worth pointing out that many adapter sequences are very similar to each other so you may get a hit reported which isn't technically correct, but which has very similar sequence to the actual match.
+
+With DNA sequencing data no single sequence should be present at a high enough frequency to be listed, but we can sometimes see a small percentage of adapter reads. RNA sequencing data may have some transcripts that are so abundant that they register as overrepresented sequence.
+
+> ### {% icon details %} More details about other FastQC plots
 >
->     It plots out the percentage of base calls at each position for which an N was called.
+> #### Per tile sequence quality
+> 
+> ![Per tile sequence quality](../../images/quality-control/per_tile_sequence_quality.png "Per tile sequence quality")
+> 
+> This graph will only appear for Illumina library which retains its original sequence identifiers. Encoded in these is the flowcell tile from which each read came. The graph enables you to look at the quality scores from each tile across all of your bases to see if there was a loss in quality associated with only one part of the flowcell.
+>   
+> The plot shows the deviation from the average quality for each tile. The colours are on a cold to hot scale, with cold colours being positions where the quality was at, or above, the average for that base in the run, and hotter colours indicate that a tile had worse qualities than other tiles for that base. In the example above you can see that certain tiles show consistently poor quality. A good plot should be blue all over.
```

Comment:
I often observe something like this in our RNASeq data:

![pertile](https://user-images.githubusercontent.com/25689525/62200221-3f90fe80-b385-11e9-943d-3088fdd80aa8.png)

we analysed this a bit further and have noticed that the horizontal lines are tiles where the chemicals arrive last. That is the chemistry is a bit worse there.

Reply:
I added this example in the explanation. Let me know what you think about it

Reply:
fine for me